page-faq.php

```diff
@@ -161,13 +161,14 @@
-<ul>
-	<li><strong>Core</strong> means part of ownCloud core, like sharing, gallery, external storage, encryption etc. Being part of core, problems would block releases and the employees of ownCloud, Inc. maintain most of the components.</li>
-	<li><strong>Official</strong> are apps which are quite well maintained and important for ownCloud. They include the Calendar and Contacts. Being 'official', we deeply care about them and will try to have them available for new releases on release day, but they are still developed by volunteers. This also means there is no commercial support available from ownCloud, Inc. This often also means that these apps do not work well on systems with large number of users or large amounts of data.</li>
-	<li><strong>Approved</strong> means that an app won’t immediately eat your data, that we have made sure there are no security problems and we know the maintainers.</li>
-	<li><strong>Experimental</strong> = the rest. These apps are neither checked for stability nor security problems and you should be careful when installing them - they might compromise your privacy, security or data integrity.</li>
-</ul>
+<p>We have four types of apps: Core, Official, Approved and Experimental. To keep the official zip file of a reasonable size, we only ship the core functionality. Other apps are, however, just one click away!</p>
+<p>This is what the four categories mean:
+<ul>
+	<li><strong>Core</strong> means part of ownCloud core, like sharing, gallery, external storage, encryption. These are integral parts of the ownCloud experience and problems in these apps will block releases. ownCloud, Inc. offers support on these to customers and while not all core functionality is maintained by employees, Inc. takes responsibility for stability and of the entirety of ownCloud core.</li>
+	<li><strong>Official</strong> are apps which are quite well maintained and important for ownCloud users. They include the Calendar and Contacts. Being 'official', we deeply care about them and will try to have them available for new releases on release day, but they are still <strong>developed by volunteers</strong>. This also means there is no commercial support available from ownCloud, Inc. This often also that these apps do not work well on systems with large numbers of users or large amounts of data.</li>
```

Comment:
They include the Calendar and Contacts. -> They include the Calendar and Contacts apps.

This often also that these apps do not -> This often also means that these apps do not 
--> Could be also "This often also means that it is not guaranteed that these apps works well"